lib/3scale_toolbox/commands/service_command/service_create.rb

```diff
@@ -0,0 +1,58 @@
+module ThreeScaleToolbox
+  module Commands
+    module ServiceCommand
+      class ServiceCreateSubcommand < Cri::CommandRunner
+        include ThreeScaleToolbox::Command
+
+        def self.command
+          Cri::Command.define do
+            name        'create'
+            usage       'create [options] <remote> <service-name>'
+            summary     'service create'
+            description 'Create a service'
+            runner ServiceCreateSubcommand
+
+            param   :remote
+            param   :service_name
+
+            option :d, :'deployment-mode', "Specify the deployment mode of the service", argument: :required
+            option :s, :'system-name', "Specify the system-name of the service", argument: :required
+            option :a, :'authentication-mode', "Specify authentication mode of the service ('1' for API key, '2' for App Id / App Key, 'oauth' for OAuth mode, 'oidc' for OpenID Connect)", argument: :required
+          end
+        end
+
+        def run
+          remote = arguments[:remote]
+          client = threescale_client(remote)
+          create_service_params = service_params
+          result = client.create_service create_service_params
+          result = Entities::Service.create(client, create_service_params)
+          puts "Service #{arguments['service_name']} has been create with ID: #{result['id']}"
+        end
+
+        private
+
+        def parse_options
+          result = {}
+          if opts[:'deployment-mode']
+            result[:deployment_option] = opts[:'deployment-mode']
+          end
+          if opts[:'system-name']
+            result[:system_name] = opts[:'system-name']
+          end
+          if opts[:'authentication-mode']
+            result[:backend_version] = opts[:'authentication-mode']
+          end
+          result
+        end
+
+        def service_params
+          service_name = arguments[:service_name]
+          create_service_params = parse_options
+          create_service_params.merge!({name: service_name})
```

Comment:
small detail, but, why not
`create_service_params[:name] = service_name`

Reply:
:+1: